crates/holochain/src/conductor/conductor.rs

```diff
@@ -215,9 +220,6 @@ pub struct Conductor {
     /// the dynamically allocated port later.
     admin_websocket_ports: RwShare<Vec<u16>>,
 
-    /// Collection app interface data, keyed by id
-    app_interfaces: RwShare<HashMap<AppInterfaceId, AppInterfaceRuntime>>,
```

Comment:
All this ended up holding was one end of the signal channel that is now encapsulated in the `AppBroadcast`, so it is no longer required.

Reply:
What bothers me about this though is that the bound ports must be getting pushed into the conductor state, where we ought to be maintaining `0`s if that's how the conductor was configured. Otherwise we wouldn't be able to report the current ports.

So I think we're only binding to `0` once, then on subsequent startups we're assuming that the same ports are free. That is sort of reasonable but probably not what the user intended when they configured Holochain. Unless I'm missing something that is!

Reply:
Hmm, yeah not sure that's even sort of reasonable. Ports could easily be used up by some other process when it comes time to re-bind them.

Reply:
Cool, I'll follow up this PR by taking a closer look at this then to both make sure I'm right about how that's working and ensure we don't lose the `0`s if so